packages/theme-patternfly-org/scripts/md/parseMD.js

```diff
@@ -0,0 +1,153 @@
+const path = require('path');
+const fs = require('fs-extra');
+const unified = require('unified');
+const remove = require('unist-util-remove');
+const toVfile = require('to-vfile'); // https://github.com/vfile/vfile
+const vfileReport = require('vfile-reporter');
+const yaml = require('js-yaml'); // https://github.com/nodeca/js-yaml
+const { makeSlug } = require('../../helpers/slugger');
+const { extractTableOfContents } = require('../../helpers/extractTableOfContents');
+
+const outputBase = path.join(process.cwd(), `src/generated`);
+
+function toReactComponent(mdFilePath, source) {
+  // vfiles allow for nicer error messages and have native `unified` support
+  const vfile = toVfile.readSync(mdFilePath);
+
+  const relPath = path.relative(process.cwd(), vfile.path);
+
+  let jsx;
+  let outPath;
+  let pageData = {};
+  let frontmatter = {};
+  let toc = [];
+
+  unified()
+    .use(require('remark-parse'))
+    .use(require('remark-frontmatter'), ['yaml'])
+    // Extract frontmatter
+    .use(() => (tree, file) => {
```

Comment:
the workflow needs tests :)

Reply:
Indeed. @dgutride do I have an extra day to write them?

Reply:
I opened #1994 for this.